app/Http/Controllers/JobController.php

```diff
@@ -248,7 +248,11 @@ function ($skillLevel) use ($skillLangs) {
         return view(
             'manager/job_create',
             [
-                'job_heading' => Lang::get($jobHeading),
+                /*Localization Strings*/
+                'job_l18n' => Lang::get('manager/job_create'),
```

Comment:
This is incredibly nitpicky, but `i18n` is for internationalization, and `l10n` is for localization.

Reply:
I :heart: :nerd: nitpicky! I will fix it in a commit tomorrow 😄 .